patches/0001-Enable-building-with-C-11-and-C-14.patch

```diff
@@ -0,0 +1,91 @@
+From 451dd27fb6c8a0ff69b31e316e110df48c78086a Mon Sep 17 00:00:00 2001
```

Comment:
I think it would be better if each patch filename was prefixed with the subpackage name (and perhaps even version). That would make this stuff a lot easier to maintain.
What does the flatpak policy say?

Reply:
No policy, it's up to the maintainer. Your suggestion totally makes sense though. :+1: